README.md

```diff
@@ -290,6 +291,7 @@ Options for the `Builder` class
     * `xmldec.standalone` standalone document declaration: true or false
   * `doctype` (default `null`): optional DTD. Eg. `{'ext': 'hello.dtd'}`
   * `headless` (default: `false`): omit the XML header. Added in 0.4.3.
+  * `cdata` (default: `false`): wrap text nodes in `<![CDATA[ ... ]]>` 
```

Comment:
Well, I'd say to just use the CDATA wrapping if the text to be wrapped contains any of `&`, `<` and `>`. These are a candidate for the use of `options.cdata`, otherwise the vaulue still just get escaped by `&amp;`, `&lt;`  & `&gt;`.